demo/forked_node.sh

```diff
@@ -8,10 +8,10 @@
 #    Error HH604: Error running JSON-RPC server: error:0308010C:digital envelope routines::unsupported
 # export NODE_OPTIONS=--openssl-legacy-provider
 
-RPC_URL=https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161
+: "${RPC_URL:=https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161}"
```

Comment:
Why is this needed? From what I gather `:` always returns exit code 0, but I expect an assignment to a constant to also return exit code 0.

Reply:
Or I can change it to `FOO=${FOO:-someDefaultValue}` form which is maybe more intuitive?

Reply:
I think changing it to the more familiar form would be good.

Maybe add a comment in the top description of these scripts saying which variables can be overriden in the environment?

Reply:
SGTM, will do!

Reply:
Pushed, please take a look.